CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+* Select All functionality added to MultiSelect component
```

Comment:
Final nitpick - Can you move this into the New Features section and then I reckon this is ready to merge.